docs/src/layouts/MainLayout.vue

```diff
@@ -1,74 +1,80 @@
 <template>
   <q-layout view="hHh lpR fff" class="bg-lp-dark">
-
-    <q-header class="bg-lp-dark">
-      <q-toolbar>
+    <q-header class="bg-lp-dark horizontal-glow-bar">
+      <q-toolbar class="q-pa-md row justify-between">
         <q-btn flat @click="showDrawer = !showDrawer" round dense icon="menu" v-if="$q.screen.xs" color="lp-primary" />
-        <q-toolbar-title :class="$q.screen.xs? 'row justify-center items-center':''" class="add-vertical-bar position-relative">
-          <q-avatar>
-            <q-icon name="img:homepage-icons/quasar-logo-blue.svg" size="lg"/>
-          </q-avatar>
-          QUASAR
-        </q-toolbar-title>
-        <q-space v-if="$q.screen.gt.xs"/>
+        <div :class="$q.screen.xs? 'row justify-center items-center':''" class="add-vertical-bar position-relative">
+          <picture class="quasar-logo cursor-pointer" @click="$router.push({name: 'home'})">
+            <source srcset="https://github.com/quasarframework/quasar-art/blob/master/dist/svg/logo-dark.svg?raw=true" media="(min-width:810px) and (max-width:1080px)" width="48" height="48">
```

Comment:
When into tablet mode, the logo-dark should be centered, while it's currently aligned to the left

Reply:
Instead it's in mobile mode that the logo gets centered. In tablet mode the logo changes, but remains left aligned

Reply:
I mean it should be centered vertically and horizontally within the small box at left in which it's enclosed
![image](https://user-images.githubusercontent.com/10036108/147130649-27a6ab6f-c859-4a0d-bed5-1b3c8fd0a2b0.png)
![image](https://user-images.githubusercontent.com/10036108/147130671-a6325c71-0dbb-497f-94ff-40559d64eb5a.png)